agenda/data/src/main/java/com/gkp/agenda/data/workers/UpdatedItemsWorker.kt

```diff
@@ -32,11 +37,9 @@ class UpdatedItemsWorker(
             val agendaItem = localAgendaDataSource.getAgendaItemById(agendaItemId)
             performedRemoteUpdate(agendaItem)
             updatedItemsDao.deleteById(agendaItemId)
-            println(" ATMS UpdatedItemsWorker: success")
             Result.success()
```

Comment:
Would it make more sense to only clear the item from the database when the remote call succeeded?